TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/ResultsToLogsConsumerTest.java

```diff
@@ -87,7 +98,9 @@ public void testOutputterWithNulls() throws IOException {
         var emptyTuple = new SourceTargetCaptureTuple(rootContext.getTestTupleContext(),
                 null, null, null, null, null, null);
         try (var closeableLogSetup = new CloseableLogSetup()) {
-            var consumer = new TupleParserChainConsumer(new ResultsToLogsConsumer());
+            var resultsToLogsConsumer = new ResultsToLogsConsumer();
+            resultsToLogsConsumer.tupleLogger = closeableLogSetup.testLogger;
```

Comment:
It always bothered me that this could be set after the instance was created.  Would you like to make it an argument for a c'tor override?

Reply:
Fixed